diff --git a/src/main/java/com/windhoverlabs/yamcs/tctm/SerialATTmTcDatalink.java b/src/main/java/com/windhoverlabs/yamcs/tctm/SerialATTmTcDatalink.java
index d0f95a3..b40e6e3 100644
--- a/src/main/java/com/windhoverlabs/yamcs/tctm/SerialATTmTcDatalink.java
+++ b/src/main/java/com/windhoverlabs/yamcs/tctm/SerialATTmTcDatalink.java
@@ -54,11 +54,11 @@ public class SerialATTmTcDatalink extends AbstractLink implements Runnable, Aggr
       throw new ConfigurationException("Invalid Parity (NONE, EVEN, ODD, MARK or SPACE)");
     }
 
-    if (this.flowControl != "NONE"
-        && this.flowControl != "RTS_CTS"
-        && this.flowControl != "XON_XOFF") {
-      throw new ConfigurationException("Invalid Flow Control (NONE, RTS_CTS, or XON_XOFF)");
-    }
+    //    if (this.flowControl != "NONE"
+    //        || this.flowControl != "RTS_CTS"
+    //        || this.flowControl != "XON_XOFF") {
+    //      throw new ConfigurationException("Invalid Flow Control (NONE, RTS_CTS, or XON_XOFF)");
+    //    }
 
     if (this.dataBits != 5 && this.dataBits != 6 && this.dataBits != 7 && this.dataBits != 8) {
       throw new ConfigurationException("Invalid Data Bits (5, 6, 7, or 8)");
